src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -132,11 +132,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
 
     with self.argument_context('acr run') as c:
         c.argument('registry_name', options_list=['--registry', '-r'])
-        c.positional('source_location', help="The local source code directory path (e.g., './src') or the URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') or a remote tarball (e.g., 'http://server/context.tar.gz').", completer=FilesCompleter())
+        c.positional('source_location', help="The local source code directory path (e.g., './src') or the URL to a git repository (e.g., 'https://github.com/Azure-Samples/acr-build-helloworld-node.git') or a remote tarball (e.g., 'http://server/context.tar.gz'). /dev/null will be its value for a contextless run.", completer=FilesCompleter())
         c.argument('file', options_list=['--file', '-f'], help="The task template/definition file path relative to the source context.")
         c.argument('values', help="The task values file path relative to the source context.")
         c.argument('set_value', options_list=['--set'], help="Value in 'name[=value]' format.", action='append', validator=validate_set)
         c.argument('set_secret', help="Secret value in 'name[=value]' format.", action='append', validator=validate_set_secret)
+        # Overwrite default shorthand of cmd to make availability for image usage
+        c.argument('cmd', options_list=['--__cmd__'])
+        c.argument('cmd_value', help="Indicates whether the source location be interpreted as an image name to run or a context.", options_list=['--cmd'])
```

Comment:
@northtyphoon Think help message can be improved here to make it more generic. I might be a little out of context here but afair `--cmd` is to take any cmdlet value right and not just an image name as per yaml definition?